drivers/volume/portworx/schedops/schedops.go

```diff
@@ -29,8 +29,10 @@ type Driver interface {
 	GetServiceEndpoint() (string, error)
 	// UpgradePortworx upgrades portworx to the given docker image and tag
 	UpgradePortworx(image, tag string) error
-	//IsPXReadyOnNode returns true if PX pod is up on that node, else returns false
+	// IsPXReadyOnNode returns true if PX pod is up on that node, else returns false
 	IsPXReadyOnNode(n node.Node) bool
+	// IsPXInstalled returns true is px/enable flag is disabled/not applied on node
```

Comment:
nit: return true if portworx is enabled on given node
keep it generic

also can we rename this method to IsPxEnabled() instead of IsPxInstalled()